amee/WallVisualization[] walls
amee/Pose robotPose
amee/Point[] currentMeasurements
amee/Point[] tags